### PR TITLE
prometheus-keylight-exporter: init at 0.1.1

### PR DESCRIPTION
**Pull request by mdlayher**
###### Motivation for this change

- New Prometheus exporter for Elgato Key Light devices
- Pushing my package upstream so I don't have to maintain an overlay locally.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

```
[matt@servnerr-3:~/src/nixpkgs]$ nix-build -A prometheus-keylight-exporter
these derivations will be built:                                                                 
  /nix/store/p7yr2jywg2jv3dyxqsb4wd5dyppi226s-keylight-exporter-0.1.1.drv                                                                                                                          
building '/nix/store/p7yr2jywg2jv3dyxqsb4wd5dyppi226s-keylight-exporter-0.1.1.drv'...
...
/nix/store/7i3ydp9rysjbscwp2czzp8qkgypxifqd-keylight-exporter-0.1.1
                                                                                                 
[matt@servnerr-3:~/src/nixpkgs]$ ./result/bin/keylight_exporter -h
Usage of ./result/bin/keylight_exporter:
  -metrics.addr string
        address for Elgato Key Light exporter (default ":9288")
  -metrics.path string
        URL path for surfacing collected metrics (default "/metrics")

[matt@servnerr-3:~/src/nixpkgs]$ ./result/bin/keylight_exporter
2020/06/08 10:19:51 starting Elgato Key Light exporter on ":9288"
```

### CODE REVIEW

**Comment by mdlayher** [comment]
@GrahamcOfBorg build prometheus-keylight-exporter

File: pkgs/servers/monitoring/prometheus/keylight-exporter.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "keylight-exporter";
+  version = "0.1.1";
+
+  goPackagePath = "github.com/mdlayher/keylight_exporter";
```
Comment (zowoq) [inline pkgs/servers/monitoring/prometheus/keylight-exporter.nix:7]: Not needed with `buildGoModule`.
```suggestion
```

**Review by zowoq — APPROVED** [review]
One nit, otherwise lgtm. Builds/runs on darwin.

**Comment by mdlayher** [comment]
@zowoq Thanks!

**Review by Ma27 — CHANGES_REQUESTED** [review]
Please add a NixOS module as well when packaging a new prometheus exporter. Details on how to do that can be found here: https://nixos.org/nixos/manual/index.html#module-services-prometheus-exporters-new-exporter

**Review by mmilata — APPROVED** [review]
Module LGTM!

**Review by WilliButz — APPROVED** [review]

**Pull request merged**